### PR TITLE
Themes!

### PR DESCRIPTION
**Pull request by brenns10**
This implements #33 - adding themes from the pelican-themes repository, making a script to install them, adding a theme chooser, and integrating them into site generation.

Things you need to change:
- No models change, however you need to update your default theme in your database.  You can do this in two ways:
  + The easy way is to flush your database and repopulate it with my updated populate script:

    ```bash
    $ cd corvid
    $ python manage.py flush
    # yes...
    $ python manage.py populate
    ```

  + The other way is to open your django shell and update the theme as follows:

    ```
    $ cd corvid
    $ python manage.py shell
    >>> from main.models import Theme
    >>> default = Theme.objects.get(title='default')
    >>> default.filepath = 'notmyidea'
    >>> default.save()
    ```

- Other than updating the default theme, you also need to get the submodule (and its submodules) in order to have the pelican-themes:

  ```bash
  $ git submodule init
  $ git submodule update
  $ cd pelican-themes
  $ git submodule init
  $ git submodule update
  ```

Once you've done that, you can (finally) populate the themes:

```bash
$ cd corvid
$ python manage.py loadthemes
```

Things I don't have (and we can evaluate their importance) include:
- Pretty names for the themes.
- Image previews for the themes.

It's also worth noting that some of the themes don't come "ready-to-roll".  They want you to do stuff like `bower install` and whatever else frontend people do these days.  So it's not quite as simple as "plug and play" all the time.  It may be necessary to go through and compile a list of themes that "just won't work" and blacklist them from populating our database.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,10 +33,33 @@ User-friendly static site generation as a service. Powered by [Pelican](http://b
...
+
+```bash
+git submodule init
+git submodule update
+# wait a little while
+cd pelican-themes
+git submodule init
+git submodule update
+# wait a lot longer
+cd ../corvid
```
Comment (jpcjr) [inline README.md]: loadthemes?

File: README.md [hunk 2]
```diff
@@ -34,10 +33,33 @@ User-friendly static site generation as a service. Powered by [Pelican](http://b
 ### Packages
 - With virtualenv active, install required packages:
   `pip install -r requirements.txt`
+- This may fail due to lack of development headers when installing native
+  extensions (particularly psycopg2 and lxml).  So then you'll probably want
+  `sudo apt-get install python3-dev libxml2-dev`.
```
Comment (jpcjr) [inline README.md]: need to add `libxslt1-dev lib32z1-dev` to this list. Otherwise, lxml won't compile.

**Comment by brenns10** [comment]
Ok, did a lot of good stuff here.  Now it's no longer "choose theme", it's "project settings".  I hooked it up to the project settings link in Kyle's new project home page and it works great.

**Comment by jpcjr** [comment]
- Remove the 'e' at the top of choose_theme.html
- rename `choose_theme` template to `project_settings`. Doesn't seem to be used, and would make maintenance confusing.

**Pull request merged**